network/network.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-block_worker: http://one.devnet-0chain.net/dns
+block_worker: http://dev.0chain.net/dns
```

Comment:
Not network.yaml, please rename to config.yaml

Reply:
sure ,can i also know what network to keep by default in config.yaml should it be dev.zus.network or dev.0chain.net